### ISSUE TITLE
Make all functions to take into account the selected Dataset

### ISSUE THREAD

**Issue by henrinie** [Signbank/FinSL-signbank#159]
There are many functions that should take into account the Dataset the user is working on. For example all the AJAX functions should only return stuff related to the selected Dataset.

Before we can start to implement these changes, we need to figure out a way to make the user select a Dataset. Depending on how many are available, maybe we should set it so that it always chooses the one available, if only one is available. Store the selection of dataset to session perhaps?

Related to #129

**Comment by henrinie** [comment]
Added dataset functionality to ELAN ecv export, see commit 6513f0e.
Now users can export ECV's based on a Dataset.

**Comment by henrinie** [comment]
We have the following things done:
* ECV's are separated according to Dataset.
* The search page works properly, you can select Dataset(s) you want to search from.
* Add gloss page works, there is a Dataset selection that is required.
* In django admin you can filter based on Dataset.
* In each gloss_detail page we show which Dataset the gloss is part of.

Things we need to do:
* Figure out what to do to the menu-panel search, it has no option to choose a Dataset
    * Maybe we could make it possible to choose a Dataset in the baselayout? Users are probably going to search from one Dataset most of the time.
* In the search page we have no way to distinguish Glosses based on Datasets, if user decides to search from multiple Datasets at once.
* All the advanced gloss properties are unchecked, not sure if they need changing.
    * Not going to waste time on that since we don't need them yet.

**Comment by henrinie** [comment]
Things that have been implemented in addition to things already done:
- You can choose dataset(s) in the search feature
- In search results we are showing dataset for glosses
- In gloss page we show the dataset the gloss is part of

The only thing not implemented from the previous list is
- Figure out what to do to the menu-panel search, it has no option to choose a Dataset

**Comment by juzza81** [comment]
Thanks for those implements. Hope we can soon start to test with several datasets!
